src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,53 @@
+import Image from 'next/image';
+import spacSpaceLogo from '../../../public/icons/spacSpaceLogo.svg';
+
+export default function Footer() {
+  return (
+    <footer className="relative bottom-0 flex flex-col w-full h-[32.4rem] text-[#3F3F3F] font-[500] pl-[8rem] bg-[url('/icons/footerBackground.svg')]">
+      <Image src={spacSpaceLogo} alt="spacspace" className="mt-[6rem]" />
+      <span className="text-[2rem] font-[600] mt-[3.7rem]">CONTACT</span>
+      <div className="flex justify-between">
+        <div className="mt-[0.8rem] flex">
+          <div className="flex flex-col gap-[0.4rem]">
+            <div className="flex">
+              <span className="text-[1.6rem] text-[#969696]">(주)스팩스페이스</span>
+              <div className="flex ml-[2.6rem]">
+                <span className="text-[1.6rem] text-[#969696]">대표자</span>
+                <span className="text-[1.6rem] ml-[1.1rem]">염민호</span>
+              </div>
+            </div>
+            <span className="text-[1.6rem]">서울 강서구 마곡중앙2로 11, 3층 303호</span>
+            <div>
+              <span className="text-[1.6rem] text-[#969696]">Email</span>
+              <span className="text-[1.6rem] ml-[2.3rem]">admin@sfacspace.com</span>
```

Comment:
```suggestion
            <a href="mailto:admin@sfacspace.com" className="text-[1.6rem] ml-[2.3rem]">admin@sfacspace.com</a>
```
P5: 이메일 바로 보낼 수 있게 이렇게 하면 어떤가용?

Reply:
그게 좋겠네요. 반영하겠습니다.